src/resources/postcss/views/skeleton/_single-event.pcss

```
@@ -276,6 +307,8 @@
 	
 	/* Make sure the gmap div has this specific height on desktop */
```

Should maybe change this comment - since we're enforcing a max-height on mobile too?
```suggestion
	/* Make sure the gmap div has these specific heights */
```